ansible/roles/flux/tasks/crd_info.yml

```diff
@@ -0,0 +1,11 @@
+- name: "Verify CRD gitrepositories.source.toolkit.fluxcd.io"
```

Comment:
Is this file just for debugging purposes?

Reply:
No, it is used by delete procedure on two places. To avoid duplicated steps I take them out as separate task and just include it when needed. Custom resources with finalizers can "deadlock" and cannot be deleted.

- First the role use it to determine if the CRDs are stuck after uninstall the helm chart. If yes - patch both with empty finalizers.
- Then the role verify if the CRDs are successfully patched and deleted. If yes - delete the namespace.